docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -1657,6 +1657,34 @@ quarkus.rest-client.my-client.url=...
 
 NOTE: MicroProfile REST Client specification does not allow setting proxy credentials. In order to specify proxy user and proxy password programmatically, you need to cast your `RestClientBuilder` to `RestClientBuilderImpl`.
 
+=== Local proxy for Dev Mode
```

Comment:
I would go with lowercase as I think that's what we use everywhere but I won't block for that :).
```suggestion
=== Local proxy for dev mode
```

Reply:
Applied :)